chunky/src/java/se/llbit/chunky/renderer/renderdump/ClassicDumpFormat.java

```diff
@@ -43,7 +43,7 @@ public void readSamples(DataInputStream inputStream, Scene scene, PixelConsumer
         g = inputStream.readDouble();
         b = inputStream.readDouble();
         consumer.consume(pixelIndex, r, g, b);
-        pixelProgress.accept(pixelIndex);
+        pixelProgress.accept(progress++);
```

Comment:
Looks like the warning in line 38 also applies to the progress :smile:

Reply:
Indeed 😅